### ISSUE TITLE
Quick pick doesn't use SPAA

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#86335]
Testing https://github.com/microsoft/vscode/issues/84715

![image](https://user-images.githubusercontent.com/2193314/70190416-b5e29c00-16aa-11ea-8e54-5ab2557df926.png)

**Comment by chrmarti** [comment]
I set the list's background with https://github.com/microsoft/vscode/commit/98c14586a4d2e3a5add95c81df32f930044c8a03. Need to check if the widget's background is set. (/cc @alexandrudima )

**Comment by chrmarti** [comment]
I realize this is on QuickOpen where we attach a styler, maybe too late to fix this. I don't see a background or listBackground option on the old tree. @joaomoreno ?

**Comment by bpasero** [comment]
I would not invest in classic quick open but make sure quick pick is good.

**Comment by joaomoreno** [comment]
Yup, not gonna invest here. Let's get rid of the last tree usage.
